src/components/Events/Style.jsx

```diff
@@ -0,0 +1,7 @@
+import styled from 'styled-components';
+
+const SEvents = styled.div`
+  display: flex;
+  flex-template-columns: -50vw 50vw;
+`;
+export default SEvents;
```

Comment:
this PR concerns the modifications of the map section
I think more style is needed for this section